x/interchainstaking/keeper/callbacks_test.go

```diff
@@ -1142,7 +1143,13 @@ func TestValsetCallbackNilValidatorReqPagination(t *testing.T) {
 
 	data := []byte("\x12\"\n 00000000000000000000000000000000")
 	err := keeper.ValsetCallback(quicksilver.InterchainstakingKeeper, ctx, data, icqtypes.Query{ChainId: suite.chainB.ChainID})
-	suite.NoError(err)
+	if err != nil {
+		if err.Error() == "attempted to unmarshal zero length byte slice" {
+			fmt.Println("Successfully rejected zero lenfth byte slice")
+		} else {
+			t.FailNow()
+		}
+	}
```

Comment:
we should update these lines by `suite.Require().EqualError(err, "attempted to unmarshal zero length byte slice")`